CHANGELOG.md

```diff
@@ -1,8 +1,15 @@
 ## <sub>main</sub>
 #### _Unreleased_
 
-## <sub>v3.0.1</sub>
+## <sub>v3.0.2</sub>
 #### _Unreleased_
+**Bugfixes**
+* Fixed load error in Selenium Webdriver Edge
```

Comment:
* Fix Load error in Edge Options
  * Mac Edge was not properly processing the browser name by default